### ISSUE TITLE
provider_name needs more h-padding in Top Sites

### ISSUE THREAD

**Issue by pdehaan** [mozilla/activity-streams#195]
I've been going through a bunch of random sites from my RSS feeds and trying to get a good sample of random content (spoiler alert: favicons are almost always ugly-low-res so #184 is more common than I expected), and noticed that some site's `provider_name` get oddly cropped in the [Top Sites](https://github.com/mozilla/activity-streams/blob/0e6e8f7ccff03af23f93b1add8af1f983d4acd20/content-src/components/TopSites/TopSites.js#L27) container.

```scss
    .tile-title {
      font-size: $tile-title-font-size;
      border-radius: 0 0 $border-radius $border-radius;
      background-color: $tile-title-bg-color;
      height: $tile-title-height;
      line-height: $tile-title-height;
      text-align: center;
      overflow: hidden;
    }
```
&mdash; via [./content-src/components/TopSites/TopSites.scss:31-39](https://github.com/mozilla/activity-streams/blob/0e6e8f7ccff03af23f93b1add8af1f983d4acd20/content-src/components/TopSites/TopSites.scss#L31-L39) 

---

![new_tab](https://cloud.githubusercontent.com/assets/557895/13273204/9c34d622-da56-11e5-8630-4e0290c1f243.png)

**Comment by k88hudson** [comment]
:+1:

**Comment by pdehaan** [comment]
Fixed via #205. Closing.
